operators/src/source/gdal_source.rs

```diff
@@ -372,6 +374,155 @@ impl MetaData<GdalLoadingInfo, RasterResultDescriptor, RasterQueryRectangle>
     }
 }
 
+#[derive(PartialEq, Serialize, Deserialize, Debug, Clone)]
+#[serde(rename_all = "camelCase")]
+pub struct GdalMetadataFixedTimes {
+    pub time_steps: Vec<TimeInterval>,
+    pub params: GdalDatasetParameters,
+    pub result_descriptor: RasterResultDescriptor,
+    pub time_placeholders: HashMap<String, GdalSourceTimePlaceholder>,
+}
+
+impl GdalMetadataFixedTimes {
+    #[allow(dead_code)]
```

Comment:
remove

Reply:
I couldn't figure out how to fix the warning so I changed it to `#[allow(unused)]`

Reply:
This method is only used externally for instantiating the struct and not used within `loading_info.rs`, thus it is unused.